modules/gke-cluster/variables.tf

```diff
@@ -53,44 +53,9 @@ variable "binary_authorization_enforcement_mode" {
   default = "ENFORCED_BLOCK_AND_AUDIT_LOG"
 }
 
-variable "binary_authorization_admission_whitelist_pattern_0" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_1" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_2" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_3" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_4" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_5" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_6" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
-}
-
-variable "binary_authorization_admission_whitelist_pattern_7" {
-  # This value cannot be empty, so use a placeholder value that satisfies Google's API.
-  default = "PLACE.HOLDER/PATTERN"
+variable "binary_authorization_admission_whitelist_patterns" {
+  # This list cannot be empty, so use a placeholder value that satisfies Google's API.
+  default = ["PLACE.HOLDER/PATTERN"]
```

Comment:
Is this still true - that the list can't be empty?

Reply:
Yes.

Reply:
Can you elaborate on why please? I don't think I understand.

Reply:
I added the error message to the comment.